core/src/env_platform/env_service.bal

```diff
@@ -1,11 +1,12 @@
 import ballerina/http;
-import ballerina/openapi;
+// import ballerina/openapi;
+// import ballerina/io;
 
 listener http:Listener ep0 = new (9090);
 
-@openapi:ServiceInfo {
-    contract: "resources/openapi_v3.yaml"
-}
+// @openapi:ServiceInfo {
```

Comment:
Any reason why these were commented out? If they are not needed anymore we can remove them.

Reply:
Could you please check with the latest commits I have resolved it.